Makefile

```diff
@@ -10,3 +14,11 @@ test:
 
 pre-commit:
 	@pre-commit run --all-files
+
+.PHONY: helm-push
+helm-push:
+	@for chart in $$(find $(CHARTS_DIR) -maxdepth 2 -type f -name "Chart.yaml" | cut -d "/" -f 3); do \
```

Comment:
"-maxdepth 2" only cover the top level examples, we should also release the helm charts for components, which are in the helm-charts/common directory.
https://github.com/opea-project/GenAIInfra/tree/charts-release